ccf_transaction_processor/CMakeLists.txt

```diff
@@ -35,6 +35,13 @@ add_ccf_app(pdoenc
   INCLUDE_DIRS ${CCF_DIR}/include/ccf ${CCF_DIR}/include/3rdparty ${CCF_DIR}/include}
 )
 
-install(TARGETS pdoenc.virtual DESTINATION lib)
+# sign and install target
+IF ("virtual" IN_LIST COMPILE_TARGETS)
+  install(TARGETS pdoenc.virtual DESTINATION lib)
+ENDIF()
 
-sign_app_library(pdoenc.enclave ../etc/oe_sign.conf ${PDO_ENCLAVE_CODE_SIGN_PEM})
+IF ("sgx" IN_LIST COMPILE_TARGETS)
```

Comment:
Seems a bit confusing to use `virtual` vs `sgx` whereas in other contexts it is `virtual` vs `release` (e.g., `expand_config`) or `SIM` vs `HW` (traditional SGX/PDO world).    But then maybe this is a CCF and/or OE thing?